getting-started/02-new-cif-project.md

```diff
@@ -0,0 +1,394 @@
+
+It is also considered a best practice to include the Adobe Public Maven Repository in your Maven settings. This file, commonly found at `~/.m2/settings.xml` can be updated following this [Knowledge Base](https://helpx.adobe.com/experience-manager/kb/SetUpTheAdobeMavenRepository.html) article.
+
+## Generate the project
+
+The next series of steps will take place using a UNIX based command line  terminal,  but should be similar if using a Windows terminal.
+
+1. Verify that Maven is correctly installed and available from the command line:
+
+    ```shell
+    $ mvn -version
+      Apache Maven 3.3.9
+      Maven home: /Library/apache-maven-3.3.9
+      Java version: 11.0.2, vendor: Oracle Corporation
+      Java home: /Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home
+    ```
+
+2. Navigate to a directory, i.e a source folder, in which you want to generate the project:
+
+    ```shell
+    $ cd ~/src
+    ```
+
+3. Copy+Paste the following command to initiate the creation of a new project in interactive mode:
+
+    ```shell
+    mvn archetype:generate \
+    -DarchetypeGroupId=com.adobe.commerce.cif \
+    -DarchetypeArtifactId=cif-project-archetype \
+    -DarchetypeVersion=0.2.0
+    ```
+
+    > *Note, in the above command we use version **0.2.0** of the archetype. As a best practice, always use the latest [released version of the archetype](https://github.com/adobe/aem-cif-project-archetype/releases).*
+
+4. The CIF Project Archetype will prompt a series of parameters to generate the project. For the sake of this tutorial and in order to see the effects of the different parameters we will generate the project for the fictitious **Acme Corporation**.
+
+    Use the following values:
+
+    | Name                  | Value                | Description                        |
+    | --------------------- | -------              | ---------------------------------- |
+    | groupId               | **com.acme.cif**     | Base Maven groupId                 |
+    | appsFolderName        | **acme**             | `/apps` folder name                |
+    | artifactId            | **acme-store**       | Base Maven ArtifactId              |
+    | version               | **1.0-SNAPSHOT**     | Version                            |
+    | package               | **com.acme.cif**     | Java Source Package                |
+    | artifactName          | **Acme Store**       | Maven Project Name                 |
+    | componentGroupName    | **Acme.Content**     | AEM component group name           |
+    | confFolderName        | **acme**             | `/conf` folder name                |
+    | contentFolderName     | **acme**             | `/content` folder name             |
+    | optionAemVersion      | *6.5.0*              | Target AEM version                 |
+    | optionIncludeExamples | *y*                  | Include Component Library examples |
```

Comment:
> Include Component Library examples
I know it's copied from the archetype readme, but it's actually wrong. The `optionIncludeExamples` option includes samplecontent as sample pages. See https://github.com/adobe/aem-cif-project-archetype/tree/master/src/main/archetype/samplecontent.